### PR TITLE
React: Don't stop after the first match

### PR DESCRIPTION
**Pull request by salto-edit**

### CODE REVIEW

File: modules/react.py [hunk 1]
```diff
@@ -61,10 +61,11 @@ def sendmsg(source, recip, text):
         return "Vuoi pure che ti dica grazie? Gli altri ti odieranno per quello che hai fatto."
 
     text = text.lower()
-    for i in messages:
-        if text.rfind(i) != -1:
-            return messages[i]
-    return None
+    values = ()
```
Comment (ltworf) [inline modules/react.py]: What you want here is `values = []`

**Comment by ltworf** [comment]
@vlopezj how will our users react to the change in interface, given that the bot will start to react to multiple reacts after this change?

**Comment by vlopezj** [comment]
I think they can deal with it :)

ltworf notifications@github.com skrev: (13 maj 2015 16:53:54 CEST)

**Comment by ltworf** [comment]
Hm, are you sure it's a good idea? Maybe they will be confused and will not know what is going on.

**Comment by vlopezj** [comment]
They can ask for a refund if they don't like it.

ltworf notifications@github.com skrev: (13 maj 2015 17:14:18 CEST)

File: modules/react.py [hunk 2]
```diff
@@ -61,10 +61,11 @@ def sendmsg(source, recip, text):
...
+    for k in messages:
+        if text.rfind(k) != -1:
```
Comment (ltworf) [inline modules/react.py]: Actually this is quite ugly, it can be replaced by `if k in text:` I guess I'll do it in the next commit if I don't forget.

**Comment by salto-edit** [comment]
I fixed the issues mentioned in your line comments. Please merge.

**Comment by ltworf** [comment]
@salto-edit thanks for the fixes!

The list is needed instead of a tuple because tuples are not mutable and hashable objects in Python, while lists are mutable (and unhashable).

As a more stylistic note, for the next time it would have been better to have just 2 commits with the proper initializations, you could use git rebase to do that.

I'm going to merge and test this.

**Comment by salto-edit** [comment]
Sure, but won't that mess up the pull request? Maybe not... will try next time.

**Comment by ltworf** [comment]
You need to rewrite history with a push -f :) But be careful not to destroy everything when you do it.

**Pull request merged**